components/pedigree/resources/src/main/resources/pedigree/pedigree.js

```diff
@@ -68,6 +73,11 @@ define([
         CausalGeneLegend,
         RejectedGeneLegend,
         RejectedCandidateGeneLegend,
+        CandidateNovelDiseaseGeneLegend,
+        CandidateNovelPhenGeneLegend,
+        UmcGeneLegend,
+        UmcVusGeneLegend,
+        UmcMsvGeneLegend,
```

Comment:
There HAS to be a better way. Adding a few gene status values should not require so much code being added to the pedigree editor to keep up. If it does, then there's a major design flaw.